spec/features/user/user_profile_spec.rb

```diff
@@ -47,12 +48,14 @@
     click_on "Edit Profile"
 
     fill_in :username, with: "StringsNThings"
+    select 'Teacher', from: 'Role'
     click_on "Submit Changes"
 
     expect(page).to have_content("Username has already been taken")
     expect(current_path).to eq("/users/#{@user.id}/edit")
 
     fill_in :username, with: "Bruce Banner"
+    select 'Teacher', from: 'Role'
```

Comment:
would be interested to make sure that editing a user profile and changing the role successfully updates the user profile with the new role and removes the old role in a test.